### PR TITLE
[WFCORE-6047] Upgrade Jakarta JSON Processing to 2.1.1 and Eclipse Pa…

### PR DESCRIPTION
**Pull request by yersan**
…rsson to 1.1.1

Jira issue: https://issues.redhat.com/browse/WFCORE-6047

### CODE REVIEW

**Review by bstansberry — DISMISSED** [review]

**Comment by yersan** [comment]
Hi @bstansberry , this PR Jira was blocked by https://github.com/wildfly/wildfly/pull/16063. I've just kicked off the integration Jobs again

**Comment by yersan** [comment]
@bstansberry tests are green, so I'll take your dismissed approval as an approval

**Pull request merged**
